LFS171x/sawtooth-material/sawtooth-tuna/client/src/state.js

```diff
@@ -0,0 +1,91 @@
+'use strict'
```

Comment:
Provide license information for this file.
`// SPDX-License-Identifier: Apache-2.0`